content/en/blog/_posts/2024-07-xx-kube-scheduler-simulator/index.md

```diff
@@ -0,0 +1,137 @@
+
+This debuggable scheduler can also run standalone, e.g., on any Kubernetes cluster or in integration tests. 
+This would be useful to custom plugin developers who want to test their plugins or examine their custom scheduler in a real cluster with better debuggability.
+
+## The simulator as a better dev cluster
+
+As mentioned earlier, with a limited set of tests, it is impossible to predict every possible scenario in a real-world cluster.
+Typically, you will test the scheduler in a small, development cluster before deploying it to production, hoping that no issues arise.
+
+[The simulator’s importing feature](https://github.com/kubernetes-sigs/kube-scheduler-simulator/blob/master/simulator/docs/import-cluster-resources.md)
+provides a solution by allowing you to simulate deploying a new scheduler version in a production-like environment without impacting your live workloads.
+
+By continuously syncing between your production cluster and the simulator, you can safely test a new scheduler version with the same resources your production cluster handles. 
+Once confident in its performance, you can proceed with the production deployment, reducing the risk of unexpected issues.
+
+## What are the use cases?
+
+1. **Cluster users**: Examine if your scheduling constraints (e.g., PodAffinity, PodTopologySpread) work as intended.
+2. **Cluster admins**: Assess how your cluster would behave with changes to the scheduler configuration.
+3. **Scheduler plugin developers**: Test your custom scheduler plugins or extenders, use the debuggable scheduler in integration tests or development clusters, and use [the syncing feature](https://github.com/kubernetes-sigs/kube-scheduler-simulator/blob/master/simulator/docs/import-cluster-resources.md) for testing within a production-like environment. 
+
+## Getting started
+
+The simulator only requires Docker to be installed on your machine; a Kubernetes cluster is not necessary.
+
+```
+git clone git@github.com:kubernetes-sigs/kube-scheduler-simulator.git
+cd kube-scheduler-simulator
+make docker_up
+```
+
+You can then access the simulator's web UI at http://localhost:3000.
+
+Visit the [kube-scheduler-simulator repository](https://sigs.k8s.io/kube-scheduler-simulator) for more details!
+
+## Getting involved 
+
+The Kube-scheduler-simulator is developed by Kubernetes SIG Scheduling. Your feedback and contributions are welcome!
+
+Open issues or PRs at [Kube-scheduler-simulator repository](sigs.k8s.io/kube-scheduler-simulator)
+Join the conversation on the [#sig-scheduling](https://kubernetes.slack.com/messages/sig-scheduling) slack channel.
+
+## Acknowledgments
+
+The simulator has been maintained by dedicated volunteer engineers, overcoming many challenges to reach its current form. 
+
+A big shout out to all [the awesome contributors](https://github.com/kubernetes-sigs/kube-scheduler-simulator/graphs/contributors)!
+
+![contributors](./contributors.png)
```

Comment:
This actually doesn't need a `figure` shortcode, but _should_ have a more descriptive `alt` attribute.